apps/happy-blocks/package.json

```diff
@@ -21,11 +21,12 @@
 		"build:support-content-links": "calypso-build --env block='support-content-links'",
 		"build-translations-manifest": "yarn run build-calypso-strings && node bin/build-translations-manifest.js",
 		"clean": "rm -r release-files block-library/*/build || true",
-		"dev": "yarn run calypso-apps-builder --localPath / --remotePath /home/wpcom/public_html/wp-content/a8c-plugins/happy-blocks"
```

Comment:
I'd ultimately want to keep calypso apps builder and simply rework it. It really just does two things:
1. Run every `build:*` script in the calypso app (many apps have multiple entrypoints for different blocks or features). These build scripts normally just use calypso-build.
2. Setup support for the `--sync` commands for wpcom sandboxes, which i think is important to keep.

Perhaps this means making esbuild etc an option for `calypso-build`.

Thank you for the exploration!!